helper/storagepacker/storagepacker.go

```diff
@@ -347,7 +347,7 @@ func NewStoragePacker(view logical.Storage, logger log.Logger, viewPrefix string
 	packer := &StoragePacker{
 		view:         view,
 		viewPrefix:   viewPrefix,
-		logger:       logger,
+		logger:       logger.Named("storagepacker"),
```

Comment:
I think we're better off letting whatever is creating the storage packer specify the logger name, in case a given backend (say, token store) ends up having more than one storage packer implementation. But we could move to that later if needed, so it's fine to keep this.